kumascript/macros/LearnSidebar.ejs

```diff
@@ -799,29 +808,29 @@ var text = mdn.localStringMap({
         'Advanced_text_formatting': 'Продвинутое форматирование текста',
         'Document_and_website_structure': 'Структура документа и веб-сайта',
         'Debugging_HTML': 'Отладка HTML',
-        'Assessment_marking_up_a_letter': 'Задание: Выделение символа',
-        'Assessment_structuring_a_page_of_content': 'Задание: Структура страницы',
+        'Assessment_Marking_up_a_letter': 'Задание: Выделение символа',
+        'Assessment_Structuring_a_page_of_content': 'Задание: Структура страницы',
       'Multimedia_and_embedding': 'Мультимедиа и встраивание',
         'Multimedia_and_embedding_overview': 'Мультимедиа и встраивание',
         'Images_in_HTML': 'Изображения в HTML',
         'Video_and_audio_content': 'Видео и аудио контент',
-        'Other_embedding_technologies': 'От object до iframe — другие технологии встраивания',
-        'Adding_vector_graphics_to_the_Web': 'Добавление векторный графики в Веб',
+        'From_object_to_iframe_other_embedding_technologies': 'От object до iframe — другие технологии встраивания',
+        'Adding_vector_graphics_to_the_web': 'Добавление векторный графики в Веб',
         'Responsive_images': 'Отзывчивые изображения',
         'Assessment_Mozilla_splash_page': 'Задание: Страница о Mozilla',
       'HTML_tables' : 'HTML таблицы',
         'HTML_tables_overview' : 'HTML таблицы',
         'HTML_table_basics' : 'HTML таблицы основы',
-        'HTML_table_advanced' : 'HTML таблицы продвинутые возможности и доступность',
+        'HTML_table_advanced_features_and_accessibility' : 'HTML таблицы продвинутые возможности и доступность',
         'Assessment_Structuring_planet_data' : 'Задание: Структурирование данных о планетах',
-    'CSS_styling_the_Web': 'CSS — стилизирование Веба',
+    'CSS_Styling_the_web': 'CSS — стилизирование Веба',
       'CSS_first_steps': 'Введение в CSS',
         'CSS_first_steps_overview': 'Введение в CSS',
         'What_is_CSS':  'Что такое CSS?',
         'Getting_started_with_CSS': 'Начало работы с CSS',
         'How_CSS_is_structured': 'Как структурирован CSS',
         'How_CSS_works': 'Как работает CSS',
-        'Using_your_new_knowledge': 'Использование ваших новых знаний',
+        'Assessment_Styling_a_biography_page': 'Assessment: Styling a biography page',
```

Comment:
@myshov Based on https://developer.mozilla.org/ru/docs/Learn/CSS/First_steps/Styling_a_biography_page, the translation should remain the same:

```suggestion
        'Assessment_Styling_a_biography_page': 'Использование ваших новых знаний',
```